grammars/javascript.cson

```diff
@@ -363,6 +363,10 @@
     'name': 'support.constant.dom.js'
   }
   {
+    'match': '(?<=\\.)([a-zA-Z_?.$][\\w?.$]*)\\b'
```

Comment:
I'm a little unclear why `?` is in these character classes, I didn't think you could have question marks in property names.